server/lib/celluloid_tracer.rb

```diff
@@ -0,0 +1,17 @@
+module CelluloidCallTracer
+  def method_missing(method, *args, &block)
```

Comment:
(you should usually have respond_to_missing? when adding method_missing. also there's stdlib's [SimpleDelegator](https://ruby-doc.org/stdlib-2.2.1/libdoc/delegate/rdoc/SimpleDelegator.html) for this kind of thing)